sw/device/tests/alert_handler_lpg_clkoff_test.c

```diff
@@ -440,6 +446,8 @@ bool test_main(void) {
   size_t test_step_cnt;
   size_t peri_idx;
 
+  flash_ctrl_testutils_backdoor_init(&flash_ctrl);
+
   // Need a NVM counter to keep the test-step info
   // between resets on the FPGA.
   if (kDeviceType == kDeviceFpgaCw310) {
```

Comment:
You can remove the ` flash_ctrl_testutils_default_region_access()` here as it is now called inside the `flash_ctrl_testutils_backdoor_init(&flash_ctrl)` function above.

https://github.com/lowRISC/opentitan/blob/c68ba96e0eb37402da8c570dc36c8be5878129be/sw/device/tests/alert_handler_lpg_clkoff_test.c#L454-L462